### PR TITLE
Update Chinese lang. Closes #758

### LINKED ISSUE #758: Chinese langs
Hey I have just translated the new strings recently. Can you guys kick off the build and make a sync? Thanks

### PR DESCRIPTION
**Pull request by wfjsw**
Please Wait until all problem fixed.

### CODE REVIEW

**Comment by wfjsw** [comment]
OK now

**Comment by jbrooksuk** [comment]
Thanks!

File: resources/lang/zh-CN/reminders.php [hunk 1]
```diff
@@ -1,14 +1,5 @@
 <?php
 
-/*
```
Comment (GrahamCampbell) [inline resources/lang/zh-CN/reminders.php]: ?

Comment (jbrooksuk) [inline resources/lang/zh-CN/reminders.php]: It'll be CrowdIn. I'll re-add.

**Pull request merged**